### DISCUSSION TITLE
Not getting `z foo<SPACE><TAB>` completions

### DISCUSSION THREAD

**Discussion by rieje** [ajeetdsouza/zoxide#897] [Q&A]
I'm not getting `z foo<SPACE><TAB>` completions and I'm not sure why. Latest versions Zsh 5.9, Zoxide 0.9.6.

My .zshrc is:
    ...
    compinit
    ...
    source /usr/share/fzf/completion.zsh
    FZF_ALT_C_COMMAND= source /usr/share/fzf/key-bindings.zsh
    ...
    eval "$(zoxide init zsh)"
    alias z='__zoxide_z'

and I also removed `zcompdump`. 

I run:

    z /usr/bin
    z ~/bin
    z bin<SPACE><TAB>     <- here, I expect completions for both the paths have none

I'm on Arch and the path to the completions is already in `fpath`. 

P.S. Unrelated, but anyone know of a tool similar to zoxide but doesn't record frequency? I would rather manually add a list of of entries and order by priority (this should make it more predictable). I could probably script something up but for a tool that's used as frequent as `cd`, I want to be sure it's the right solution and is efficient and complete with completions which I'm not sure how to write myself but can probably tweak from examples.

**Comment by ajeetdsouza** [comment]
What is this line for?

```zsh
alias z='__zoxide_z'
```

**Reply by rieje** [reply]
I had an alias `z` already, I was just using this to override my alias to test out zoxide. 

I found the culrprit--I'm using [fzf-tab-completion](https://github.com/lincheney/fzf-tab-completion) and commenting out sourcing its plugin works. I tried sourcing that plugin before and after `eval "$(zoxide init zsh)"` but could not get both to work. Are they simply incompatible?
